cmd/suggest.go

```diff
@@ -42,6 +44,14 @@ func suggestFixesOpenAPI(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	level, err := cmd.Flags().GetString("level")
+	if err != nil {
+		return err
+	}
+	if !slices.Contains([]string{"error", "warn", "info"}, level) {
```

Comment:
may be better to use `errors.Severity` enum here instead of magic strings: https://github.com/speakeasy-api/openapi-generation/blob/main/pkg/errors/errors.go#L12

Reply:
Great point